src/actions.js

```diff
@@ -237,8 +243,8 @@ export const startWallet = (words, pin) => async (dispatch) => {
   walletUtil.cleanLoadedData();
 
   const networkName = 'mainnet';
-  const firstAddress = getFirstAddressFromWords(words, networkName);
```

Comment:
Can we remove `getFirstAddressFromWords` method?

Reply:
Yes we can, done in 0ed36ff319aa50ca4e96c0c564c5e7d3f27eca30